### PR TITLE
feat: gmp amplifier

### PR DESCRIPTION
**Pull request by ffe9f8**
Based on the [Amplifier e2e example](https://www.notion.so/bright-ambert-2bd/Guide-Amplifier-e2e-example-a15f6cc33691434d83a93ab5a5b7e1e4) Notion doc. Written in a tutorial format.

See outstanding comments.

Preview: https://axelar-docs-git-marty-amplifier-gmp-axelar-network.vercel.app/validator/amplifier/gmp-tutorial

Duplicated in [PR 841](ttps://github.com/axelarnetwork/axelar-docs/pull/841).

### CODE REVIEW

File: src/pages/validator/amplifier/gmp-tutorial.mdx [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+```
+
+### Relay the execute data to the destination chain gateway
+
+The execute data can be relayed to the destination chain’s `AxelarGateway` in a number of ways. Here’s how to do it with `node.js` using `ethers`:
```
Comment (ffe9f8) [inline src/pages/validator/amplifier/gmp-tutorial.mdx:83]: Can we list the other ways?

File: src/pages/validator/amplifier/gmp-tutorial.mdx [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+## Prerequisites
+
+You will need:
+
+- An [`axelard` node](/validator/amplifier/integrator-onboarding) that is [connected to the devnet](/validator/amplifier/verifier-onboarding)
+- A [Solidity contract](https://github.com/cjcobb23/axelar-examples/blob/main/examples/evm/call-contract/CallContract.sol) deployed to the Ethereum Sepolia and Avalanche Fuji testnets
+
+## Helpful references
+
+- [Amplifier devnet contract addresses](https://github.com/axelarnetwork/axelar-contract-deployments/pull/187/files)
```
Comment (ffe9f8) [inline src/pages/validator/amplifier/gmp-tutorial.mdx:14]: Currently links to a PR, should have direct link to GitHub JSON file once that PR is submitted.

File: src/pages/validator/amplifier/gmp.mdx [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+> await wallet.sendTransaction({to: gatewayAddress, data: "0x"+executeData, gasLimit:ethers.BigNumber.from(5e+06)}).then((tx) => tx.wait())
+```
+
+### Execute the contract
+
+Fill in the appropriate values and execute the contract. The `command_id` can be found in the response to the `get_proof` query:
+
+```js
+> await contract.execute(ethers.utils.arrayify(command_id),source_chain, source_address, ethers.utils.arrayify(payload)).then((tx) => tx.wait()
+```
```
Comment (ffe9f8) [inline src/pages/validator/amplifier/gmp.mdx:95]: Could also include what events / messages are emitted on success/failure on these transactions, so error handling by integrators can be done properly.

File: src/pages/validator/amplifier/gmp.mdx [hunk 4]
```diff
@@ -0,0 +1,95 @@
...
+
+### Relay the execute data to the destination chain gateway
+
+The execute data can be relayed to the destination chain’s `AxelarGateway` in a number of ways. Here’s how to do it with `node.js` using `ethers`:
```
Comment (ffe9f8) [inline src/pages/validator/amplifier/gmp.mdx:83]: List other ways

File: src/pages/validator/amplifier/gmp.mdx [hunk 5]
```diff
@@ -0,0 +1,95 @@
+# General Message Passing with Amplifier
+
+In this tutorial, you will learn how to execute a GMP call from the Ethereum Sepolia to the Avalanche Fuji testnets.
```
Comment (StephenFluin) [inline src/pages/validator/amplifier/gmp.mdx:3]: I'd love some additional description about why this is so different than normal GMP, or why we are doing all of this manual work. Ideally we'd justify the additional complexity and talk about the benefits.

File: src/pages/validator/amplifier/gmp.mdx [hunk 6]
```diff
@@ -0,0 +1,95 @@
...
+```
+
+The output should look something like this:
+
+```bash
+data:
+  data:
+    commands:
+    - id: c767e0ff9bcfcfa11d5346095b2c969fedb1c4a87bbf9fd0cb82578eda6fecc2
+      params: 00000000000000000000000000000000000000000000000000000000000000a000000000000000000000000000000000000000000000000000000000000000e000000000000000000000000079a35fd62dbaa820d3025b3400ebce17cdd6c58764b8427717b2ce8573a0c37e03e30ec683c2fe57dcee4d426d22cf43c769267500000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000010657468657265756d2d7365706f6c696100000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000002a30784134663130663736423836453031423938646146363641336430326136356531346164623037363700000000000000000000000000000000000000000000
```
Comment (StephenFluin) [inline src/pages/validator/amplifier/gmp.mdx:69]: Should we truncate these giant numbers for legibility?

File: src/pages/validator/amplifier/gmp.mdx (hunk 5 above)
Comment (cjcobb23) [reply src/pages/validator/amplifier/gmp.mdx:3]: it's not really different than normal GMP. These steps ultimately will be performed by the relayer, not the user. This tutorial is mainly for people building relayers.

**Pull request closed without merging**